conformance/utils/tester/tester.go

```diff
@@ -0,0 +1,38 @@
+package tester
+
+import (
+	"testing"
+)
+
+// Tester allows users to add custom logging shims to conformance tests.
+type Tester interface {
+	testing.TB
+
+	// Run calls a subtest and wraps the testing.T.Run call
+	Run(name string, f func(Tester))
+
+	// Parallel should pass through to testing.T.Parallel
+	Parallel()
+}
+
+// Impl performs basic wrapping of testing.T
+type Impl struct {
```

Comment:
Do you need to export Impl?

Reply:
Not really, I can unexport if preferred.